packages/awsfirehose/changelog.yml

```
@@ -1,9 +1,14 @@
 # newer versions go on top
+- version: "1.8.1"
+  changes:
+    - description: Update correct link for pull request
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/14691
 - version: "1.8.0"
   changes:
     - description: Updates the ingest pipeline to use a more efficient script for setting event.dataset matching
       type: enhancement
-      link: https://github.com/elastic/integrations/pull/14691
+      link: https://github.com/elastic/integrations/pull/13910
```

We should keep this one with the correct link right? 
```suggestion
      https://github.com/elastic/integrations/pull/14691
```